validator/attester/service.go

```diff
@@ -58,8 +61,9 @@ func NewAttester(ctx context.Context, cfg *Config) *Attester {
 // Start the main routine for an attester.
 func (a *Attester) Start() {
 	log.Info("Starting service")
-	client := a.rpcClientService.AttesterServiceClient()
-	go a.run(a.ctx.Done(), client)
+	attester := a.rpcClientService.AttesterServiceClient()
+	validator := a.rpcClientService.ValidatorServiceClient()
+	go a.run(a.ctx.Done(), attester, validator)
```

Comment:
no need to pass in a.ctx.Done() here as the routine can access it as a property of the struct